package.json

```diff
@@ -7,7 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "./node_modules/.bin/webpack -d",
     "lint": "./node_modules/.bin/eslint app/* server.js; exit 0",
-    "start": "nodemon server.js"
+    "start": "nodemon server.js",
+    "knex": "knex"
```

Comment:
this points at a global knex module. Instead have it point at the module in this app.
`./node_modules/.bin/knex`